compare/src/components/molecules/ApproveButton.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { approveTest, filterTests } from '../../actions';
+import { colors, fonts } from '../../styles';
+
+const REMOTE_HOST = 'http://127.0.0.1';
+const REMOTE_PORT = 3000;
```

Comment:
There is an [open issue](https://github.com/garris/ember-backstop/issues/34) in `ember-backstop` to make backstop run on a port other than the default port 3000. Will having this as a constant make it harder to implement that ?

Reply:
This is a great point.   When Backstop opens a report (either internally or via user on cli) it runs openReport.js -- notice here is where the port is defined...
https://github.com/garris/BackstopJS/blob/724f16e76ae353b7b37cc18915c053d03b73c2af/core/command/openReport.js#L9
So, it is probably just a matter of pulling out the dynamic port property out of the config at that point and passing into the above port definition.

AN ADDITIONAL THING TO NOTE:  This also happens to be where we explicitly tell the report UI that it is in REMOTE mode.   Some features, like `Approve` will only work in remote mode (i.e. when backstopRemote is running).  I don't remember if the UI has a global variable to track this -- but it needs one.  And the `Approve` option should not be shown unless we are in remote mode.

Reply:
@1276stella ☝️